app/src/main/java/org/glucosio/android/tools/TimeElapsed.java

```diff
@@ -0,0 +1,45 @@
+package org.glucosio.android.tools;
+
+import java.util.Calendar;
+
+/**Singleton Class to intiate start time and calculate the time elapsed
+ * Created by Viney Ugave on 10/25/16.
+ */
+
+public class TimeElapsed {
+
+    private static TimeElapsed _instance = null;
+    private static long startTime;
+    private static long endTime;
+
+    protected TimeElapsed() {
+        // Exists only to defeat instantiation.
```

Comment:
If you want really prevent class creation you should make it final and constructor private